pyproject.toml

```diff
@@ -40,7 +40,7 @@ Source = "https://github.com/opensafely-core/job-runner"
 branch = true
 dynamic_context = "test_function"
 source = ["jobrunner"]
-omit = ["*jobrunner/cli/manifests.py", 'jobrunner/service.py']
+omit = ["jobrunner/cli/manifests.py", 'jobrunner/service.py']
```

Comment:
There was 0% coverage of `cli/manifests`, I assume the intent of this was to ignore the file, but it wasn't working?